reference/3.0/Microsoft.PowerShell.Management/Set-Service.md

```diff
@@ -1,75 +1,91 @@
 ---
-ms.date:  06/09/2017
+ms.date:  07/17/2019
 schema:  2.0.0
 locale:  en-us
 keywords:  powershell,cmdlet
 online version: https://go.microsoft.com/fwlink/?linkid=113399
 external help file:  Microsoft.PowerShell.Commands.Management.dll-Help.xml
 title:  Set-Service
 ---
+
 # Set-Service
 
 ## SYNOPSIS
-
 Starts, stops, and suspends a service, and changes its properties.
 
 ## SYNTAX
 
 ### Name (Default)
 
 ```
-Set-Service [-Name] <String> [-ComputerName <String[]>] [-DisplayName <String>] [-Description <String>]
- [-StartupType <ServiceStartMode>] [-Status <String>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
+Set-Service [-Name] <String> [-ComputerName <String[]>] [-DisplayName <String>]
+ [-Description <String>] [-StartupType <ServiceStartMode>] [-Status <String>] [-PassThru] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
 ```
 
 ### InputObject
 
 ```
 Set-Service [-ComputerName <String[]>] [-DisplayName <String>] [-Description <String>]
- [-StartupType <ServiceStartMode>] [-Status <String>] [-InputObject <ServiceController>] [-PassThru] [-WhatIf]
- [-Confirm] [<CommonParameters>]
+ [-StartupType <ServiceStartMode>] [-Status <String>] [-InputObject <ServiceController>] [-PassThru]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-The `Set-Service` cmdlet changes the properties of a service.
-This includes the status, description, display name, and start mode.
-You can use this cmdlet to start, stop, or suspend, or pause, a service.
-To identify the service, enter its service name or submit a service object, or pipe a service name or service object to `Set-Service`.
+The `Set-Service` cmdlet changes the properties of a service such as the status, description,
```

Comment:
If you don't mean the PowerShell properties, which I don't think you do. I would choose a different word, like attributes.  Otherwise I think you mean $Service.Status which means it should be bolded.  

Either way is fine. You can just bold Status, Description, etc. or use attributes instead of properties

Reply:
Fixed.